python/cuml/solvers/qn.pyx

```diff
@@ -206,8 +206,9 @@ class QN(Base,
         will not be regularized.
     max_iter: int (default = 1000)
         Maximum number of iterations taken for the solvers to converge.
-    tol: float (default = 1e-3)
-        The training process will stop if current_loss > previous_loss - tol
+    tol: float (default = 1e-4)
+        The training process will stop if
+          `norm(current_loss_grad, inf) <= tol * current_loss`
```

Comment:
It would be helpful to provide a very short summary here for users (as you've done in this PR already) just to clarify how this differs from Scikit-learn.